api/processor_api.py

```diff
@@ -303,7 +347,21 @@ async def update_config(config: ConfigDTO):
             success = restart_processor()
             return handle_config_response(config_dict, success)
 
-        @app.post('/area')
+        @app.get("/areas")
+        async def list_areas():
+            return {
+                "areas": get_areas()
+            }
+
+        @app.get("/areas/{area_id}")
+        async def get_area(area_id):
+            areas = get_areas()
+            filtered_areas = list(filter(lambda area: area['id'] == area_id, areas))
```

Comment:
Instead of filter you should get the `first()` that matches a condition. You can do it like this:
https://stackoverflow.com/a/9868665/4708657